src/clr-angular/popover/common/abstract-popover.ts

```diff
@@ -90,6 +92,13 @@ export abstract class AbstractPopover implements AfterViewChecked, OnDestroy {
         return this.ifOpenService.open ? false : true;
     }
 
+    @HostListener("keydown", ["$event"])
+    onKeyDown(event: KeyboardEvent) {
```

Comment:
This event listener is not on the document, so it only triggers if the key is pressed while focus is inside of the popover. Is that what we want?
I don't have an opinion, but whichever the decision is, please add a comment explaining it or linking to the corresponding conversation on GitHub.

Reply:
I don't believe dropdowns or tooltips should have a focus trap, unless I missed something. So you can't rely on that.
As I said, I'm fine either way, I just want the decision to be clear and explicitly commented. :)

Reply:
ok. i will check with @reddolan once. Thanks @youdz